packages/frontend/core/src/modules/workbench/view/route-container.css.ts

```diff
@@ -4,7 +4,7 @@ import { style } from '@vanilla-extract/css';
 export const root = style({
   display: 'flex',
   height: '100%',
-  overflow: 'hidden',
+  overflow: 'clip',
```

Comment:
what's the difference visually

Reply:
no difference visually, but hidden can still be scrolled by the program, while clip cannot be scrolled by the program.
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#clip